idem_azurerm/states/azurerm/redis/operations.py

```diff
@@ -284,6 +301,29 @@ async def present(
         )
 
     if "error" not in cache:
+        if wait_for_ready and action == "create":
+            if check_interval < 30 or check_interval > 600:
+                log.error(
```

Comment:
if you see a bad config, but fix it for the user and still proceed... use `log.warning`